podcasts/PodcastImageView.swift

```diff
@@ -44,9 +44,9 @@ class PodcastImageView: UIView {
     }
 
     func setTransparentNoArtwork(size: PodcastThumbnailSize) {
+        imageView?.kf.cancelDownloadTask()
```

Comment:
I added this because if the ImageView was being used for a async loading of image, we could have a scenario where we set the image to nil, but meanwhile the download finishes and we set it to the the podcast image previously being loaded.

Reply:
I see that this method is only used in `FolderPreviewView.swift` so we should be fine?

Reply:
Even with only that use this can happen, because we know are reusing the image views for folder,  so if you set to transparent, but on the previous use of the FolderPreviewView it was being loaded we need to cancel that download.